workshop/pre-work/README.md

```diff
@@ -3,41 +3,32 @@
 Before we get started, we will download some assets and complete some setup for our workshop. This section is broken up into the following steps:
 
 - [Pre-work](#pre-work)
-  - [1. Download or Clone the Repository](#1-download-or-clone-the-repository)
+  - [1. Download the data](#1-download-the-data)
```

Comment:
`Data` should be in Upper case for consistency? (They are all Camel

Reply:
done